### PR TITLE
Ccg liberty boot #1207809 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/20464 - where you can find a link to the rendered output (Client Configuration Guide).
~https://bugzilla.suse.com/show_bug.cgi?id=1207809~ [ATM, no longer reproducible.]

Use salt macro.
# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/administration/pages/custom-channels.adoc [hunk 1]
```diff
@@ -79,6 +79,7 @@ endif::[]
 ifeval::[{uyuni-content} == true]
 If you are registering traditional {rhel}{nbsp}7 clients you might notice errors with unsigned packages.
 endif::[]
+
```
Comment (keichwa) [inline modules/administration/pages/custom-channels.adoc:82]:
```suggestion
```

**Comment by keichwa** [comment]
I added the initial report as a comment to bootstrap-repository.adoc.  Maybe, we also need it there.

**Review by 0rnela — APPROVED** [review]

File: modules/client-configuration/pages/bootstrap-repository.adoc [hunk 2]
```diff
@@ -161,3 +161,41 @@ Automatic creation of a bootstrap repository might fail if you are using custom
 In this case, you need to create the repository manually.
 
 For more information about custom channels, see xref:administration:custom-channels.adoc[].
+
+
+
+=== Create Bootstrap Repository for SUSE Liberty Linux 9
```
Comment (0rnela) [inline modules/client-configuration/pages/bootstrap-repository.adoc:167]:
```suggestion
=== Create Bootstrap Repository for {sll} 9
```

File: modules/client-configuration/pages/bootstrap-repository.adoc [hunk 3]
```diff
@@ -161,3 +161,41 @@ Automatic creation of a bootstrap repository might fail if you are using custom
...
+copy 'venv-salt-minion-3004-1.3.3.x86_64'
+Directory walk started
+Directory walk done - 1 packages
+Temporary output repo path: /srv/www/htdocs/pub/repositories/res/9/bootstrap.tmp/.repodata/
+Preparing sqlite DBs
+Pool started (with 5 workers)
+Pool finished
+////
+
+When creating the bootstrap repository for SUSE Liberty Linux 9 the user must select the Salt bundle contact method.
```
Comment (0rnela) [inline modules/client-configuration/pages/bootstrap-repository.adoc:187]:
```suggestion
When creating the bootstrap repository for {sll} 9 the user must select the Salt bundle contact method.
```

**Comment by keichwa** [comment]
@mcalmer I'd appreciate if you or your team could review this!

File: modules/client-configuration/pages/bootstrap-repository.adoc [hunk 4]
```diff
@@ -161,3 +161,41 @@ Automatic creation of a bootstrap repository might fail if you are using custom
...
+copy 'venv-salt-minion-3004-1.3.3.x86_64'
+Directory walk started
+Directory walk done - 1 packages
+Temporary output repo path: /srv/www/htdocs/pub/repositories/res/9/bootstrap.tmp/.repodata/
+Preparing sqlite DBs
+Pool started (with 5 workers)
+Pool finished
+////
+
+When creating the bootstrap repository for {sll} 9 the user must select the Salt bundle contact method.
```
Comment (keichwa) [inline modules/client-configuration/pages/bootstrap-repository.adoc:187]:
```suggestion
When creating the bootstrap repository for {sll} 9 the user must select the {salt} bundle contact method.
```

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 5]
```diff
@@ -40,6 +40,8 @@ endif::[]
 Traditional clients are not available on {sll}{nbsp}8 or 9.
 {sll}{nbsp}8 and 9 clients are only supported as Salt clients.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc:41]:
```suggestion
{sll}{nbsp}8 and 9 clients are only supported as {salt} clients.
```

Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc:41]:
```suggestion
{sll}{nbsp}8 and 9 clients are only supported as {salt} clients.
```

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc [hunk 6]
```diff
@@ -28,6 +28,7 @@ If you switch, the [package]``salt-minion`` package and its dependencies will st
 
 
 
+[[contact-methods-saltbundle-using]]
 === Using the Salt Bundle with the Salt Minion
 
 The Salt Bundle can be used with the Salt Minion managed by the Salt Master other than {productname} Server at the same time.
```
Comment (keichwa) [inline modules/client-configuration/pages/contact-methods-saltbundle.adoc:34]:
```suggestion
=== Using the {salt} Bundle with the {salt} Minion

The {salt} Bundle can be used with the {salt} Minion managed by the {salt} Master other than {productname} Server at the same time.
```

**Comment by keichwa** [comment]
> Please use the attribute for SUSE Liberty Linux as suggested.

ok, thanks!

File: modules/client-configuration/pages/bootstrap-repository.adoc [hunk 7]
```diff
@@ -161,3 +161,41 @@ Automatic creation of a bootstrap repository might fail if you are using custom
...
+Temporary output repo path: /srv/www/htdocs/pub/repositories/res/9/bootstrap.tmp/.repodata/
+Preparing sqlite DBs
+Pool started (with 5 workers)
+Pool finished
+////
+
+When creating the bootstrap repository for {sll} 9 the user must select the {salt} bundle contact method.
+
+
+* With the [command]``mgr-bootstrap`` command use the [option]``--force-bundle`` parameter, or
```
Comment (mcalmer) [inline modules/client-configuration/pages/bootstrap-repository.adoc:190]: I am not sure if the --force switch is really needed.
A current bootstrap script should use venv-salt-minion "by default".
If the bootstrap repo has it, it should be used. Also without this --force switch.

If somebody saw a problem, I would like to know more about it.

Comment (jcayouette) [reply modules/client-configuration/pages/bootstrap-repository.adoc:190]: Agreed, anything with `--force` looks sketchy. :D

**Comment by 0rnela** [comment]
> > Please use the attribute for SUSE Liberty Linux as suggested.
> 
> ok, thanks!

Thank you @keichwa. Approved from my pov.

**Review by 0rnela — APPROVED** [review]

File: modules/client-configuration/pages/bootstrap-repository.adoc [hunk 8]
```diff
@@ -161,3 +161,41 @@ Automatic creation of a bootstrap repository might fail if you are using custom
...
+Creating bootstrap repo for SUSE-LibertyLinux9-x86_64
+copy 'venv-salt-minion-3004-1.3.3.x86_64'
+Directory walk started
+Directory walk done - 1 packages
+Temporary output repo path: /srv/www/htdocs/pub/repositories/res/9/bootstrap.tmp/.repodata/
+Preparing sqlite DBs
+Pool started (with 5 workers)
+Pool finished
+////
+
```
Comment (keichwa) [inline modules/client-configuration/pages/bootstrap-repository.adoc:186]:
```suggestion

```

File: modules/client-configuration/pages/bootstrap-repository.adoc (hunk 7 above)
Comment (keichwa) [inline modules/client-configuration/pages/bootstrap-repository.adoc:190]:
```suggestion
```

File: modules/client-configuration/pages/bootstrap-repository.adoc [hunk 9]
```diff
@@ -161,3 +161,41 @@ Automatic creation of a bootstrap repository might fail if you are using custom
...
+* Modify the bootstrap script manually and set:
+----
+# Avoid installing venv-salt-minion instead salt-minion
+# even if it available in the bootstrap repo
+AVOID_VENV_SALT_MINION=0
+
+# Force installing venv-salt-minion instead salt-minion
+# even if it is NOT available in the bootstrap repo
+FORCE_VENV_SALT_MINION=1
+----
```
Comment (keichwa) [inline modules/client-configuration/pages/bootstrap-repository.adoc:201]:
```suggestion
```

File: .changelog [hunk 10]
```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Improved description about creating bootstrap repository for
+  SUSE Liberty Linux in Client Configuration Guide (bsc#1207809))
```
Comment (keichwa) [inline .changelog:8]:
```suggestion
```

**Comment by keichwa** [comment]
bug withdrawn.  Now it is just a cosmetic issue.

**Pull request merged**